wifite/tools/airmon.py

```diff
@@ -339,6 +339,14 @@ def terminate_conflicting_processes():
         """ Deletes conflicting processes reported by airmon-ng """
         airmon_output = Process(['airmon-ng', 'check']).stdout()
 
+        # Checking for systemd, otherwise assume openrc
+        
+        if os.path.exists('/usr/lib/systemd/systemd'):
```

Comment:
The proper way would be

```
try:
    subprocess.check_output(['systemctl', '--version'])
    init_system = 'systemd'
except FileNotFoundError:
    init_system = 'openrc'
```

yet that requires importing subprocess python lib

No reason to experiment when quick and dirty way works just fine